nucypher/blockchain/eth/token.py

```diff
@@ -383,14 +416,13 @@ def sync(self) -> None:
         stake_info = self.staking_agent.get_substake_info(staker_address=self.staker_address,
                                                           stake_index=self.index)  # < -- Read from blockchain
 
-        first_period, last_period, locked_value = stake_info
-        if not self.first_locked_period == first_period:
+        if not self.first_locked_period == stake_info.first_period:
             raise self.StakingError("Inconsistent staking cache.  Make sure your node is synced and try again.")
 
         # Mutate the instance with the on-chain values
-        self.final_locked_period = last_period
-        self.value = NU.from_nunits(locked_value)
-        self.worker_address = self.staking_agent.get_worker_from_staker(staker_address=self.staker_address)
+        self.final_locked_period = stake_info.last_period
+        self.value = NU.from_nunits(stake_info.locked_value)
+        self._status = None
```

Comment:
Does this indicate an *unknown* status?

Reply:
`_status` is only for caching purposes, so it's more not yet calculated status